ui/src/app/machines/views/MachineList/MachineListTable/MachineListTable.js

```diff
@@ -83,6 +83,25 @@ const getGroupSecondaryString = (machineIDs, selectedIDs) => {
   return string;
 };
 
+/**
+ * Filters columns by hiddenColumns.
+ *
+ * If showActions is true, the "fqdn" column will not be filtered.
+ * @param {Array} columns - headers or rows
+ * @param {string[]} hiddenColumns - columns to hide, e.g. ["zone"]
+ * @param {bool} showActions
+ */
+const filterColumns = (columns, hiddenColumns, showActions) => {
+  if (hiddenColumns.length === 0) {
+    return columns;
+  }
+  return columns.filter(
+    (column) =>
+      !hiddenColumns.includes(column.key) |
```

Comment:
Did you mean double `||`?

Reply:
err yes, I blame typescript. How is this working?

Reply:
Haha yeah I have no idea.